src/components/Calendar.js

```diff
@@ -67,6 +68,10 @@ class Calendar extends React.Component {
       this.props.selectedDate || this.props.minimumDate || moment().unix()
   };
 
+  componentDidMount() {
+    moment.locale(this.props.locale)
```

Comment:
I'm assuming we don't need to import the locale files from moment because that is happening in `raja` now?

Reply:
Unfortunately we do. The locale does not get picked up in mx-react-components from Serenity, strangely. But Serenity _does_ pick up the locale from mx-react-components, which is why we needed to set that at the top level in Serenity.

Reply:
Sorry, I misunderstood your initial question - I thought it was about `componentDidMount()`. I assume we don't need to import those, because it's all working... 🤔

Reply:
Ok. In that case I think this MR is good to go.